src/menu.tsx

```diff
@@ -151,6 +152,15 @@ export function get_menu_items(
                 },
                 { type: "divider" },
             );
+        if (item.type === 'reg' || item.type === "dir")
+            menuItems.push(
+                {
+                    id: "create-symlink",
```

Comment:
This is now the first or second entry in the menu. It feels more like a fringe feature to me, how about we move that to the bottom?

Reply:
Sure, I've asked Garrett for the placement but never got a reply.

Reply:
Like this ?
![image](https://github.com/user-attachments/assets/131b0396-3c0c-4129-941d-e5f66a53268a)